apps/web/lib/supabase-queries.ts

```diff
@@ -265,6 +265,8 @@ export function usePostgresChanges(
 export const postgresSubscriptionsAtom = atom<PostgresSubscription[]>([])
 postgresSubscriptionsAtom.onMount = (setAtom) => {
   const invalidate = () =>
+    // TODO: this needs to be passed in per request otherwise it's shared
+    // across requests during SSR
```

Comment:
@tonyxiao I might need your help to fix this part.

Reply:
Missed this earlier. Will look into it now.